glam/api/views.py

```diff
@@ -439,3 +443,71 @@ def random_probes(request):
         probes.append({"data": agg.histogram, "info": probe.info})
 
     return Response({"probes": probes})
+
+
+def log_probe_query(request):
+    query = request.data["query"]
+    UsageInstrumentation(
+        action_type=UsageInstrumentation.ACTION_PROBE_SEARCH,
+        context=query,
+        tracking_id=request.COOKIES.get("session"),
+        probe_name=query["probe"],
+    ).save()
```

Comment:
It might be a good idea to add some guards around the `dict` accessors here just in case for some reason they get passed a `query` dict that doesn't have those keys. The outer `request.data.get("query")` and also the `query.get("probe")`. If they come back as `None` you can simply return without saving anything?

Reply:
I suspect if `query` or `query["probe"]` are missing from this function's parameter it'll probably be a programming error. So I'm looking for ways to help surface the error. 
Maybe I should call this function from  `get_glean_aggregations` and `get_firefox_aggregations` instead, because they both validate the query object before?